sdk/keyvault/test-resources.json

```diff
@@ -195,6 +195,10 @@
             "type": "string",
             "value": "[variables('azureKeyVaultUrl')]"
         },
+        "AZURE_KEYVAULT_HSM_URL": {
```

Comment:
Nit: technically, Managed HSM isn't Key Vault despite having a similar Keys API. Not a big deal, but I called my variable `AZURE_MANAGEDHSM_URI` (or "URL" - .NET favors "URI" but seems to be alone there).